nomic/aws/sagemaker.py

```diff
@@ -140,15 +139,18 @@ def embed_texts(
         texts: List of texts to be embedded.
         sagemaker_endpoint: The sagemaker endpoint to use.
         region_name: AWS region sagemaker endpoint is in.
-        batch_size: Size of each batch.
```

Comment:
nit: should add `task_type` to docstring

Reply:
done, thanks for catch!